l10n_br_nfe/hooks.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2019-2020 - Raphael Valyi Akretion
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+import logging
+import pkg_resources
+import nfelib
+from nfelib.v4_00 import leiauteNFe_sub as nfe_sub
+
+from odoo import api, SUPERUSER_ID
+from odoo.exceptions import ValidationError
+from odoo.addons.spec_driven_model import hooks
+
+_logger = logging.getLogger(__name__)
+
+
+def pre_init_hook(cr):
+    """
+    The objective of this hook is to ensure the Brazil country is
+    translated as "Brasil" in pt_BR to get the NFe tests pass
+    even if the pt_BR language pack is not installed.
+    """
+    cr.execute("""SELECT id
+    FROM ir_translation
+    WHERE name='res.country,name' AND
+    lang='pt_BR'""")
+    if not cr.fetchone():
+        cr.execute(
+            """
+            INSERT INTO ir_translation
+            (name, res_id, lang, type, src, value, module, state)
+            VALUES ('res.country,name', 31, 'pt_BR', 'model',
+            'Brazil', 'Brasil', 'base', 'translated');
+            """)
+
+
+def post_init_hook(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    hooks.register_hook(env, 'l10n_br_nfe',
+                        'odoo.addons.l10n_br_nfe_spec.models.v4_00.leiauteNFe')
+
+    hooks.post_init_hook(cr, registry, 'l10n_br_nfe',
+                         'odoo.addons.l10n_br_nfe_spec.models.v4_00.leiauteNFe')
+    cr.execute("select demo from ir_module_module where name='l10n_br_nfe';")
```

Comment:
Deveria evitar esse SELECT usando a chave do config: if not tools.config["without_demo"]